docs/developers/guides/run-linea/run-the-stack/local.mdx

```diff
@@ -0,0 +1,115 @@
+---
+description: Run the Linea stack locally.
+image: /img/socialCards/run-the-linea-stack-locally.jpg
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+# Run locally
+
+## Prerequisites
+
+The following installed:
+
+- [Node.js v18](https://nodejs.org/en/download/package-manager)
+- [Docker v24](https://docs.docker.com/get-docker/)
+- [Docker Compose v2.19 or later](https://docs.docker.com/compose/install/)
+- [Make v3.81 or later](https://www.npmjs.com/package/make)
+- [Pnpm >=v9.0.6](https://pnpm.io/installation)
+
+:::important
+These instructions runs the Linea stack using Docker containers. Ensure you set your Docker resources to
+use at a minimum 10 CPU cores, 8GB of memory and 1GB swap space.
+:::
+
+## Run the Linea stack
+
+### 1. Clone the repository
+
+Clone the Linea repository:
+
+```bash
+git clone https://github.com/Consensys/linea-monorepo.git
+```
+
+### 2. Install the dependencies
+
+Change to the root of the directory and install the project dependencies:
+
+```bash
+cd linea-monorepo
+```
+
+```bash
+pnpm i
```

Comment:
I know it's the same thing , but I think we should use the full `pnpm install` for consistency.

```suggestion
pnpm install
```